rotkehlchen/chain/arbitrum_one/modules/hop/constants.py

```diff
@@ -0,0 +1,48 @@
+
+
+from typing import Final
+from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
+from rotkehlchen.chain.evm.decoding.hop.constants import HopBridge
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import (
+    A_DAI_ARB,
+    A_ETH,
+    A_HOP_ARB,
+    A_MAGIC_ARB,
+    A_RETH_ARB,
+    A_USDC_ARB,
+    A_USDT_ARB,
+)
+
+
+ETH_BRIDGE: Final = string_to_evm_address('0x3749C4f034022c39ecafFaBA182555d4508caCCC')
+ETH_AMM_WRAPPER: Final = string_to_evm_address('0x33ceb27b39d2Bb7D2e61F7564d3Df29344020417')
```

Comment:
Have you check if there is some kind of registry and we can avoid hardcoding them?

Reply:
No mention of any registry in their mainnet addresses file.
https://github.com/hop-protocol/hop/blob/d842c19bacff7dbe0b2fda71763f5462a943a146/packages/sdk/src/addresses/mainnet.ts

There is something called `IBonderRegistry`, but it just has `isBonderAllowed` function.
https://github.com/hop-protocol/contracts/blob/3747b0b41defddc5b3c8ca328057847cd987c2c8/contracts/bridges/L1_Bridge.sol#L97